.gitignore

```diff
@@ -50,3 +50,9 @@ kratos/includes/kratos_version.h
 # Output of tests of meshing application (MMG)
 *.o.sol
 *.o.mesh
+
+cmake_build/
```

Comment:
You should add your personal excludes to the .git/info/exclude file in your Kratos installation, the files in the .gitignore should be common files to everyone

https://github.com/KratosMultiphysics/Kratos/wiki/How-to-make-Git-ignore-the-files-resulting-from-a-compilation-without-conflicts-in-.gitignore